### ISSUE TITLE
Question about code in MLP

### ISSUE THREAD

**Issue by joe-hu** [cauchyturing/UCR_Time_Series_Classification_Deep_Learning_Baseline#10]
Hi,
Thanks for the nice example. I am learning the codes of MLP and have a question about it. In line 59, code is “y = keras.layers.Dense(500, activation='relu')(x)”. Would ‘(x)’ be ‘(y)’? Since code in line 58 is “y = keras.layers.Dropout(0.1)(x)”

**Comment by cauchyturing** [comment]
Yes, checked-in.
